package.json

```diff
@@ -54,5 +54,8 @@
     "jest-fail-on-console": "^3.1.1",
     "prettier": "^3.0.3",
     "turbo": "^1.10.16"
+  },
+  "dependencies": {
+    "react-image": "^4.1.0"
```

Comment:
Did you mean to install this in `apps/web` instead?

`npm run i react-image --save --workspace=apps/web`